src/execution/local.rs

```diff
@@ -74,7 +83,15 @@ impl ExecutionContext {
                 builder = builder.with_batch_size(config.tui_batch_size);
             }
             AppType::FlightSQLServer => {
-                builder = builder.with_batch_size(config.flightsql_server_batch_size);
+                if config.dedicated_executor_enabled {
+                    builder = builder.with_batch_size(config.flightsql_server_batch_size);
```

Comment:
should `with_batch_size` be guarded by `dedicated_executor_enabled`? This seems like a mistake.

Reply:
it should not, thanks for catching that